pages/instructorAssessment/instructorAssessment.sql

```diff
@@ -104,6 +104,65 @@ WHERE
 ORDER BY
     aar.number;
 
+-- BLOCK assessment_score_histogram_by_date
+WITH assessment_instances_by_user_and_date AS (
+    SELECT
+        ai.user_id,
+        avg(ai.score_perc) AS score_perc,
+        date_trunc('day', date) AS date
```

Comment:
I decided to use the following instead:
date_trunc('day', date AT TIME ZONE ci.display_timezone) AS date